scripts/qa_crowbarsetup.sh

```diff
@@ -1209,16 +1209,16 @@ function onadmin_set_source_variables
             CLOUDLOCALREPOS="SUSE-Cloud-5-devel"
         ;;
         develcloud6)
-            CLOUDSLE12DISTPATH=/ibs/Devel:/Cloud:/6/images/iso
+            CLOUDSLE12DISTPATH=${cloud6_iso_path:='/ibs/Devel:/Cloud:/6/images/iso'}
             [ -n "$TESTHEAD" ] && CLOUDSLE12DISTPATH=/ibs/Devel:/Cloud:/6:/Staging/images/iso
-            CLOUDSLE12DISTISO="SUSE-OPENSTACK-CLOUD-6-$arch*Media1.iso"
+            CLOUDSLE12DISTISO=${cloud6_iso:="SUSE-OPENSTACK-CLOUD-6-$arch*Media1.iso"}
```

Comment:
The default value is duplicated here. IMHO it is enough to have it in one place.